website/ref/sys.md

```diff
@@ -0,0 +1,10 @@
+<!-- toc -->
+
+# Introduction
+
+The `sys:` module provides platform specific functions and constants.
```

Comment:
I know you put in the part about functions for consistency with other docs, but this module doesn't have any functions yet, so it's better to leave them out.

Reply:
It seemed to me that functions like `os.Geteuid()` were likely to be exposed in the near future but I see your point. There is no reason to make people think this elvish namespace provides functions until it actually does os.